Mage/src/main/java/mage/abilities/effects/common/CopyTokenEffect.java

```diff
@@ -41,8 +41,9 @@ public boolean apply(Game game, Ability source) {
         for (Ability ability : token.getAbilities()) {
             permanent.addAbility(ability, source.getSourceId(), game);
         }
-        permanent.getPower().setBoostedValue(token.getPower().getValue());
-        permanent.getToughness().setBoostedValue(token.getToughness().getValue());
+        // TODO: I am not sure if this should be getBaseValue() or getModifiedBaseValue()
```

Comment:
The `token` here is a template, not an actual permanent on the battlefield, so it doesn't matter.